image/image.go

```diff
@@ -29,30 +29,24 @@ func IDFromDigest(digest digest.Digest) ID {
 
 // V1Image stores the V1 image configuration.
 type V1Image struct {
-	// ID a unique 64 character identifier of the image
-	ID string `json:"id,omitempty"`
-	// Parent id of the image
-	Parent string `json:"parent,omitempty"`
-	// Comment user added comment
-	Comment string `json:"comment,omitempty"`
-	// Created timestamp when image was created
-	Created time.Time `json:"created"`
-	// Container is the id of the container used to commit
-	Container string `json:"container,omitempty"`
+	ID        string    `json:"id,omitempty"`        // ID is a unique 64 character identifier of the image
+	Parent    string    `json:"parent,omitempty"`    // Parent id of the image
+	Comment   string    `json:"comment,omitempty"`   // Comment is user added comment
+	Created   time.Time `json:"created"`             // Created timestamp when image was created
+	Container string    `json:"container,omitempty"` // Container is the id of the container used to commit
+
 	// ContainerConfig is the configuration of the container that is committed into the image
 	ContainerConfig container.Config `json:"container_config,omitempty"`
-	// DockerVersion specifies version on which image is built
-	DockerVersion string `json:"docker_version,omitempty"`
-	// Author of the image
-	Author string `json:"author,omitempty"`
+
+	DockerVersion string `json:"docker_version,omitempty"` // DockerVersion specifies version on which image is built
+	Author        string `json:"author,omitempty"`         // Author of the image
+
 	// Config is the configuration of the container received from the client
 	Config *container.Config `json:"config,omitempty"`
-	// Architecture is the hardware that the image is build and runs on
-	Architecture string `json:"architecture,omitempty"`
-	// OS is the operating system used to build and run the image
-	OS string `json:"os,omitempty"`
-	// Size is the total size of the image including all layers it is composed of
-	Size int64 `json:",omitempty"`
+
+	Architecture string `json:"architecture,omitempty"` // Architecture is the hardware that the image is build and runs on
```

Comment:
Why change all these? Looks to be inconsistent, because line 44 puts the documentation _above_, not _to the right_.

We may need to write down what our preferred notation is, but I'm not sure if should be in a "typo fix" pull request, because this is more than just a typo fix, and results in a lot more unrelated line-changes